wrappers/kn/README.md

```diff
@@ -0,0 +1,59 @@
+# ZXing-C++ Kotlin/Native Library
+
+## Install
+
+The easiest way to use the library is to fetch if from _mavenCentral_. Simply add
+
+```gradle
+implementation("io.github.zxing-cpp:kotlin-native:2.2.1")
+```
+
+to your `build.gradle.kts` file in the `dependencies` section of `nativeMain` source set.
+
+## Use
+
+A trivial use case looks like this:
+
+```kotlin
```

Comment:
Please replace the sample code verbatim with the following
```kotlin
import zxingcpp.BarcodeFormat
import zxingcpp.BarcodeReader
import zxingcpp.ImageFormat
import zxingcpp.ImageView

val data: ByteArray = ...    // the image data
val width: Int = ...         // the image width
val height: Int = ...        // the image height
val format = ImageFormat.Lum // ImageFormat.Lum assumes grey scale image data

val image = ImageView(data, width, height, format)
val barcodeReader = BarcodeReader().apply {
    formats = setOf(BarcodeFormat.EAN13, BarcodeFormat.QRCode)
    tryHarder = true
    maxNumberOfSymbols = 3
    // more options, see documentation
}

barcodeReader.read(image).joinToString("\n") { barcode: Barcode ->
    "${barcode.format} (${barcode.contentType}): ${barcode.text}"
}
```

Reply:
fixed in https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/7321b8766d52b996bed8e63bbbb5f1d5a58d44bc